stable/cockroachdb/README.md

```diff
@@ -48,14 +48,23 @@ This chart will do the following:
 To install the chart with the release name `my-release`:
 
 ```shell
-helm install --name my-release stable/cockroachdb
+helm install my-release stable/cockroachdb
 ```
 
-Note that for a production cluster, you are very likely to want to modify the `storage.persistentVolume.size` and `storage.persistentVolume.storageClass` parameters. This chart defaults to `100 GiB` of disk space per Pod, but you may want more or less depending on your use case, and the default PersistentVolume `storageClass` in your environment may not be what you want for a database (e.g. on GCE and Azure the default is not SSD).
+Note that for a production cluster, you will likely want to override the following parameters in [`values.yaml`](values.yaml) with your own values.
+
+- `statefulset.resources.requests.memory` and `statefulset.resources.limits.memory` allocate memory resources to CockroachDB on your pods.
+- `conf.cache` and `conf.max-sql-memory` are memory limits that we recommend setting to 1/4 of the above resource allocation. When running CockroachDB on Kubernetes, you must set these limits explicitly to avoid running out of memory.
```

Comment:
> When running CockroachDB on Kubernetes, you must set these limits explicitly to avoid running out of memory.

If a user is using helm, they must be using Kubernetes, so maybe qualifying this isn't necessary?